lib/Distribution/AgamaDevel.pm

```diff
@@ -7,7 +7,7 @@
 # for integration tests using the current state of code at GitHub for Agama.
 # Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
 
-package Distribution::Opensuse::AgamaDevel;
+package Distribution::AgamaDevel;
```

Comment:
this is AgamaDevel for sles, there is not point to unify this, they will be different in future.

Reply:
Unit tests complain about duplicate code but, np going for it.

Reply:
5b383abf040e62338fbdcf8380f91a2efe572ec5